odoo/fields.py

```diff
@@ -233,6 +232,7 @@ def _read_group_many2one_field(self, records, domain, order):
 
     string = None                       # field label
     help = None                         # field tooltip
+    invisible = False             # whether the field is invisible
```

Comment:
Misalignment panic! :wink: 
```suggestion
    invisible = False                   # whether the field is invisible
```